src/procedures/SignDisableControllerAck.ts

```diff
@@ -2,9 +2,15 @@ import { Procedure } from './Procedure';
 import { ProcedureType, ErrorCode, SignDisableControllerAckProcedureArgs } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure to sign an acknowledgement for disabling the controller of a security token
+ */
 export class SignDisableControllerAck extends Procedure<SignDisableControllerAckProcedureArgs> {
   public type = ProcedureType.SignDisableControllerAck;
 
+  /**
+   * - Sign an acknowledgement to disable the controller of the security token
```

Comment:
Sign data to confirm the intent of permanently disabling the Security Token's Controller functionality

(delete hyphen)